src/main/java/com/faforever/client/teammatchmaking/PartyMemberItemController.java

```diff
@@ -0,0 +1,166 @@
+  public Label uefLabel;
+  public Label cybranLabel;
+  public Label aeonLabel;
+  public Label seraphimLabel;
+  public Label crownLabel;
+  public HBox playerCard;
+  public ImageView playerStatusImageView;
+
+  private Player player;
+  private WeakReference<PartyMemberContextMenuController> contextMenuController = null;
+
+  @Override
+  public void initialize() {
+    clanLabel.managedProperty().bind(clanLabel.visibleProperty());
+  }
+
+  @Override
+  public Node getRoot() {
+    return playerItemRoot;
+  }
+
+  public void setMember(PartyMember member) {
+    this.player = member.getPlayer();
+
+    initializeBindings();
+    playerCard.pseudoClassStateChanged(LEADER_PSEUDO_CLASS, teamMatchmakingService.getParty().getOwner().equals(player));
+
+    playerStatusImageView.setImage(uiService.getThemeImage(UiService.CHAT_LIST_STATUS_PLAYING));
+    player.statusProperty().addListener(new WeakChangeListener<>((observable, oldValue, newValue) -> markMemberBusy(newValue)));
+    markMemberBusy(player.statusProperty().get());
+
+    selectFactionsBasedOnParty();
+  }
+
+  private void markMemberBusy(PlayerStatus status) {
+    if (status != PlayerStatus.IDLE) {
+      Platform.runLater(() -> {
+        playerStatusImageView.setVisible(true);
+        playerCard.pseudoClassStateChanged(PLAYING_PSEUDO_CLASS, true);
+        teamMatchmakingService.getPlayersInGame().add(player);
+      });
+    } else {
+      Platform.runLater(() -> {
+        playerStatusImageView.setVisible(false);
+        playerCard.pseudoClassStateChanged(PLAYING_PSEUDO_CLASS, false);
+        teamMatchmakingService.getPlayersInGame().remove(player);
+      });
+    }
+  }
+
+  private void initializeBindings() {
+    countryImageView.imageProperty().bind(createObjectBinding(() ->
+        countryFlagService.loadCountryFlag(player.getCountry()).orElse(null), player.countryProperty()));
+    avatarImageView.visibleProperty().bind(player.avatarUrlProperty().isNotNull().and(player.avatarUrlProperty().isNotEmpty()));
+    avatarImageView.imageProperty().bind(createObjectBinding(() -> Strings.isNullOrEmpty(player.getAvatarUrl()) ? null : avatarService.loadAvatar(player.getAvatarUrl()), player.avatarUrlProperty()));
+    leagueImageView.setManaged(false);
+    JavaFxUtil.bindManagedToVisible(clanLabel, avatarImageView, playerStatusImageView);
+
+    clanLabel.visibleProperty().bind(player.clanProperty().isNotEmpty().and(player.clanProperty().isNotNull()));
+    clanLabel.textProperty().bind(createStringBinding(() -> Strings.isNullOrEmpty(player.getClan()) ? "" : String.format("[%s]", player.getClan()), player.clanProperty()));
+    usernameLabel.textProperty().bind(player.usernameProperty());
+    leagueLabel.textProperty().bind(createStringBinding(
+        () -> i18n.get("leaderboard.divisionName", RatingUtil.getLeaderboardRating(player)).toUpperCase(),
+        player.globalRatingMeanProperty())); // TODO: replace this with divisionproperty once it is available
```

Comment:
Create an issue and reference it

Reply:
The leagueservice has already been merged into faf stack. I think it is just a matter of time for deployment now. Or maybe there needs to be extra work done to create the api endpoints. I am not sure.